### PR TITLE
feat: provide constants for common (network) hosts to rama-net's Host

### PR DESCRIPTION
**Pull request by NOOMA-42**
# Description
provide constants for common (network) hosts to rama-net's Host

# Related issues
#363

### CODE REVIEW

File: rama-net/src/address/host.rs [hunk 1]
```diff
@@ -18,6 +18,25 @@ pub enum Host {
...
+    pub const LOCALHOST_IPV4: Self = Self::Address(IpAddr::V4(Ipv4Addr::new(127, 0, 0, 1)));
+
+    /// Local loopback address (IPv6)
+    pub const LOCALHOST_IPV6: Self = Self::Address(IpAddr::V6(Ipv6Addr::new(0, 0, 0, 0, 0, 0, 0, 1)));
+
+    /// Local loopback name
+    pub const LOCALHOST_NAME: Self = Self::Name(Domain::from_static("localhost"));
+
+    /// Default address, not routable
+    pub const DEFAULT_IPV4: Self = Self::Address(IpAddr::V4(Ipv4Addr::new(0, 0, 0, 0)));
```
Comment (GlenDC) [inline rama-net/src/address/host.rs:32]: Can you also define the IPV6 version please?

File: rama-net/src/address/host.rs [hunk 2]
```diff
@@ -18,6 +18,25 @@ pub enum Host {
...
+
+    /// Broadcast address (IPv4)
+    pub const BROADCAST_IPV4: Self = Self::Address(IpAddr::V4(Ipv4Addr::new(255, 255, 255, 255)));
+
+    pub const EXAMPLE: Self = Self::Name(Domain::from_static("example.com"));
```
Comment (GlenDC) [inline rama-net/src/address/host.rs:37]: Given you already have a convention established here it follows that this is probably better named `EXAMPLE_NAME`

**Review by GlenDC — CHANGES_REQUESTED** [review]
Looks good, two minor requests, if that's resolve I'm happy to merge this one in!

**Comment by GlenDC** [comment]
Can you please also run locally `just fmt`, or in case you do not have `just` installed and do not want to, you can simply run this cmd:

```
cargo fmt --all
```

**Comment by NOOMA-42** [comment]
I wasn't aware that there's a nice tool like just ! Pushed another commit :) 

> Can you please also run locally `just fmt`, or in case you do not have `just` installed and do not want to, you can simply run this cmd:
> 
> ```
> cargo fmt --all
> ```

File: rama-net/src/address/host.rs [hunk 3]
```diff
@@ -18,6 +18,29 @@ pub enum Host {
...
+    /// Default address, not routable
+    pub const DEFAULT_IPV4: Self = Self::Address(IpAddr::V4(Ipv4Addr::new(0, 0, 0, 0)));
+
+    /// Default address, not routable (IPv6)
+    pub const DEFAULT_IPV6: Self = Self::Address(IpAddr::V6(Ipv6Addr::new(0, 0, 0, 0, 0, 0, 0, 0)));
+
+    /// Broadcast address (IPv4)
+    pub const BROADCAST_IPV4: Self = Self::Address(IpAddr::V4(Ipv4Addr::new(255, 255, 255, 255)));
+
+    pub const EXAMPLE_NAME: Self = Self::Name(Domain::from_static("example.com"));
```
Comment (GlenDC) [inline rama-net/src/address/host.rs:41]:
```suggestion
    /// `example.com` domain name
    pub const EXAMPLE_NAME: Self = Self::Name(Domain::from_static("example.com"));
```

**Review by GlenDC — APPROVED** [review]

**Pull request merged**